README.md

```diff
@@ -1,4 +1,76 @@
+
 # brand-assets
-Devfolio Brand Assets
 
-If you're using Devfolio's D as a profile picture and the D doesn't seem to be centered, try using `Devfolio Square/Logo-Square centred.png`
\ No newline at end of file
+  
+
+## Devfolio Brand Assets
+
+  
+
+Find everything you need to showcase the Devfolio brand.
+
+The following guidelines have been created to help other organsiations and people use our brands and assets, which include our logos, names, content and trademarks.
+
+  
+
+The following Assets are included;
+
+  
+
+1. `Logo + Text` assets involve the Devfolio 'D' logo with the 'Devfolio' text without any background.
+
+It has three variants -
+
+- `White`<img src ="Logo + Text - White/Devfolio - White.png" style="width: 100px; background-color: #2F3437; padding: 8px 10px 8px 10px; border-radius: 8px;margin-left: 20px; ">
```

Comment:
This looks kind of weird right now, would look nice if it was center aligned, and had some gap between the text and the logo
<img width="265" alt="image" src="https://user-images.githubusercontent.com/21277179/168822584-3aa546b8-48ed-4e9d-a0aa-3d7fdc67d02f.png">